### PR TITLE
Adverts amp

### PR DESCRIPTION
**Pull request by NataliaLKB**
Before:
image

After:
image

and for smaller viewports:

image

cc @johnduffell @stephanfowler

### CODE REVIEW

File: common/app/views/fragments/amp/adsStylesheet.scala.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+@@media (max-width: 500px) {
+    .amp-ad-container {
+        float: none;
+        margin-right: 0;
+    }
+
+    amp-ad.-amp-layout {
+        display: block;
+        margin: 0 auto;
+    }
```
Comment (NataliaLKB) [inline common/app/views/fragments/amp/adsStylesheet.scala.html]: I thought at this size the ad would look better in the center.

**Comment by johnduffell** [comment]
brilliant :+1:

**Comment by NataliaLKB** [comment]
I am going to update this to be a little closer to the website!

**Comment by NataliaLKB** [comment]
@johnduffell @stephanfowler updated the description with slightly updated adverts. Will deploy after lunch :).

**Comment by johnduffell** [comment]
:+1:

**Pull request merged**